### PR TITLE
Egld 000000 support

### PR DESCRIPTION
**Pull request by dragos-rebegea**

### CODE REVIEW

**Review by gabrielmatei — APPROVED** [review]

File: src/endpoints/tokens/token.service.ts [hunk 1]
```diff
@@ -838,6 +838,21 @@ export class TokenService {
...
 
+    const egldIdentifier = 'EGLD-000000';
+    const egldToken = new TokenDetailed({
+      identifier: egldIdentifier,
+      name: 'EGLD',
+      type: TokenType.FungibleESDT,
+      assets: await this.assetsService.getTokenAssets(egldIdentifier),
+      decimals: 18,
+      isLowLiquidity: false,
+      price: await this.dataApiService.getEsdtTokenPrice(egldIdentifier),
```
Comment (gabrielmatei) [inline src/endpoints/tokens/token.service.ts:849]: i think `egldIdentifier` should be replaced with `EGLD`

File: src/endpoints/tokens/token.service.ts [hunk 2]
```diff
@@ -838,6 +838,21 @@ export class TokenService {
       token => token.transactions ?? 0,
     );
 
+    const egldIdentifier = 'EGLD-000000';
+    const egldToken = new TokenDetailed({
+      identifier: egldIdentifier,
+      name: 'EGLD',
+      type: TokenType.FungibleESDT,
+      assets: await this.assetsService.getTokenAssets(egldIdentifier),
```
Comment (gabrielmatei) [inline src/endpoints/tokens/token.service.ts:846]: same here

**Review by gabrielmatei — CHANGES_REQUESTED** [review]

File: src/endpoints/tokens/token.service.ts (hunk 1 above)
Comment (dragos-rebegea) [reply src/endpoints/tokens/token.service.ts]: fixed

File: src/endpoints/tokens/token.service.ts (hunk 2 above)
Comment (dragos-rebegea) [reply src/endpoints/tokens/token.service.ts:846]: here we will keep the identifier EGLD-000000

**Review by gabrielmatei — APPROVED** [review]

**Review by cfaur09 — APPROVED** [review]

**Pull request merged**